packages/kg-parser-plugins/lib/parser-plugins.js

```diff
@@ -30,13 +30,15 @@ export function createParserPlugins(_options = {}) {
 
     // HELPERS -----------------------------------------------------------------
 
-    function _readFigCaptionFromNode(node, payload) {
-        let figcaption = node.querySelector('figcaption');
+    function _readFigCaptionFromNode(node, payload, selector = 'figcaption') {
+        let figcaption = Array.from(node.querySelectorAll(selector));
```

Comment:
is it worth pluralizing this variable so you can assume it's an array at a glance?

Reply:
Good call!! 👍